packages/app/src/components/search/SearchModal.tsx

```diff
@@ -83,7 +94,7 @@ export const SearchModal = ({ toggleModal }: { toggleModal: () => void }) => {
   });
 
   const handleSearchBarKeyDown = useCallback(
-    (e: KeyboardEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+    (e: KeyboardEvent<HTMLDivElement | HTMLTextAreaElement>) => {
```

Comment:
I think so. Because we are replacing the `InputBase` with a `TextField`, and they define a different type for the `onKeyDown` property, the event target has changed from `HTMLInputElement` to a `HTMLDivElement` element. If we do not change it to `HTMLDivElement`, we get this error:
![image](https://user-images.githubusercontent.com/6290749/233043649-a90054cd-3d2f-4e0d-aee1-a5bc26094c90.png)

Reply:
The error is self-explanatory, though we should indeed mention it in the changeset, any template for all these changeset instructions?

Reply:
Added a paragraph in changeset, let me know if wording should be different.